logging.js

```diff
@@ -192,12 +199,32 @@ export class LoggingClient {
 	}
 
 	legacyErrorBatch(legacyErrors) {
-		const logs = legacyErrors.map(({ message, source, lineno, colno, error, developerMessage }) => new LogBuilder(this._appId)
+		const remainingLegacyErrors = this._throttle(legacyErrors);
+
+		const logs = remainingLegacyErrors.map(({ message, source, lineno, colno, error, developerMessage }) => new LogBuilder(this._appId)
 			.withLegacyError(message, source, lineno, colno)
 			.withError(error)
 			.withMessage(developerMessage)
 			.withLocation()
 			.build());
 		this._logger.logBatch(logs);
 	}
+
+	_throttle(logs) {
+		if (!this._shouldThrottle) {
+			return logs;
+		}
+
+		const now = new Date().getTime();
+		return logs.filter((log) => {
+			const key = JSON.stringify(log);
+			const lastLogged = this._uniqueLogs.get(key);
+			if (lastLogged === undefined || now - lastLogged >= 60000) {
```

Comment:
Yeah, should probably store this in a `const`. Or do we need to make this configurable by having the parameter be `throttleRate` instead of a yes/no? Would only be needed if different apps would want different throttle rates.

Reply:
I'd probably keep this internal to the logger, expose it if people really want to start tuning. Also I feel like if tuning is needed we can implement a backoff algo in here. 

For maintenance it would be useful to have this set to a const with the unit in the name.